### ISSUE TITLE
Crash on login on Pixel Tablet, triggered by `android.view.InflateException`

### ISSUE THREAD

**Issue by nikclayton** [pachli/pachli-android#18]
**Describe the bug**

Running on a Pixel Tablet immediately crashes after login with the following stack trace:

```
Exception android.view.InflateException: Binary XML file line #7 in app.pachli:layout/fragment_timeline: Binary XML file line #7 in app.pachli:layout/fragment_timeline: Error inflating class <unknown>
Caused by: android.view.InflateException: Binary XML file line #7 in app.pachli:layout/fragment_timeline: Error inflating class <unknown>
Caused by: java.lang.reflect.InvocationTargetException
  at java.lang.reflect.Constructor.newInstance0
  at java.lang.reflect.Constructor.newInstance (Constructor.java:343)
  at android.view.LayoutInflater.createView (LayoutInflater.java:866)
  at android.view.LayoutInflater.createView (LayoutInflater.java:788)
  at com.android.internal.policy.PhoneLayoutInflater.onCreateView (PhoneLayoutInflater.java:58)
  at android.view.LayoutInflater.onCreateView (LayoutInflater.java:942)
  at android.view.LayoutInflater.onCreateView (LayoutInflater.java:962)
  at android.view.LayoutInflater.createViewFromTag (LayoutInflater.java:1016)
  at android.view.LayoutInflater.createViewFromTag (LayoutInflater.java:973)
  at android.view.LayoutInflater.inflate (LayoutInflater.java:667)
  at android.view.LayoutInflater.inflate (LayoutInflater.java:538)
  at app.pachli.components.timeline.TimelineFragment.onCreateView (TimelineFragment.kt)
  at androidx.fragment.app.Fragment.performCreateView (Fragment.java:3114)
  at androidx.fragment.app.FragmentStateManager.createView (FragmentStateManager.java:557)
  at androidx.fragment.app.FragmentStateManager.moveToExpectedState (FragmentStateManager.java:272)
  at androidx.fragment.app.FragmentManager.executeOpsTogether (FragmentManager.java:1943)
  at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute (FragmentManager.java:1839)
  at androidx.fragment.app.FragmentManager.execSingleAction (FragmentManager.java:1751)
  at androidx.fragment.app.BackStackRecord.commitNow (BackStackRecord.java:317)
  at androidx.viewpager2.adapter.FragmentStateAdapter.placeFragmentInViewHolder (FragmentStateAdapter.java:341)
  at androidx.viewpager2.adapter.FragmentStateAdapter.onViewAttachedToWindow (FragmentStateAdapter.java:276)
  at androidx.viewpager2.adapter.FragmentStateAdapter.onViewAttachedToWindow (FragmentStateAdapter.java:67)
  at androidx.recyclerview.widget.RecyclerView.dispatchChildAttached (RecyclerView.java:8291)
  at androidx.recyclerview.widget.RecyclerView$5.addView (RecyclerView.java:924)
  at androidx.recyclerview.widget.ChildHelper.addView (ChildHelper.java:107)
  at androidx.recyclerview.widget.RecyclerView$LayoutManager.addViewInt (RecyclerView.java:9344)
  at androidx.recyclerview.widget.RecyclerView$LayoutManager.addView (RecyclerView.java:9302)
  at androidx.recyclerview.widget.RecyclerView$LayoutManager.addView (RecyclerView.java:9289)
  at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk (LinearLayoutManager.java:1676)
  at androidx.recyclerview.widget.LinearLayoutManager.fill (LinearLayoutManager.java:1622)
  at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren (LinearLayoutManager.java:687)
  at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2 (RecyclerView.java:4604)
  at androidx.recyclerview.widget.RecyclerView.dispatchLayout (RecyclerView.java:4307)
  at androidx.recyclerview.widget.RecyclerView.onLayout (RecyclerView.java:4873)
  at android.view.View.layout (View.java:24421)
  at android.view.ViewGroup.layout (ViewGroup.java:6440)
  at androidx.viewpager2.widget.ViewPager2.onLayout (ViewPager2.java:527)
  at android.view.View.layout (View.java:24421)
  at android.view.ViewGroup.layout (ViewGroup.java:6440)
  at com.google.android.material.appbar.HeaderScrollingViewBehavior.layoutChild (HeaderScrollingViewBehavior.java:149)
  at com.google.android.material.appbar.ViewOffsetBehavior.onLayoutChild (ViewOffsetBehavior.java:43)
  at androidx.coordinatorlayout.widget.CoordinatorLayout.onLayout (CoordinatorLayout.java:918)
  at android.view.View.layout (View.java:24421)
  at android.view.ViewGroup.layout (ViewGroup.java:6440)
  at androidx.drawerlayout.widget.DrawerLayout.onLayout (DrawerLayout.java:1263)
  at android.view.View.layout (View.java:24421)
  at android.view.ViewGroup.layout (ViewGroup.java:6440)
  at android.widget.FrameLayout.layoutChildren (FrameLayout.java:332)
  at android.widget.FrameLayout.onLayout (FrameLayout.java:270)
  at android.view.View.layout (View.java:24421)
  at android.view.ViewGroup.layout (ViewGroup.java:6440)
  at android.widget.LinearLayout.setChildFrame (LinearLayout.java:1891)
  at android.widget.LinearLayout.layoutVertical (LinearLayout.java:1729)
  at android.widget.LinearLayout.onLayout (LinearLayout.java:1638)
  at android.view.View.layout (View.java:24421)
  at android.view.ViewGroup.layout (ViewGroup.java:6440)
  at android.widget.FrameLayout.layoutChildren (FrameLayout.java:332)
  at android.widget.FrameLayout.onLayout (FrameLayout.java:270)
  at android.view.View.layout (View.java:24421)
  at android.view.ViewGroup.layout (ViewGroup.java:6440)
  at android.widget.LinearLayout.setChildFrame (LinearLayout.java:1891)
  at android.widget.LinearLayout.layoutVertical (LinearLayout.java:1729)
  at android.widget.LinearLayout.onLayout (LinearLayout.java:1638)
  at android.view.View.layout (View.java:24421)
  at android.view.ViewGroup.layout (ViewGroup.java:6440)
  at android.widget.FrameLayout.layoutChildren (FrameLayout.java:332)
  at android.widget.FrameLayout.onLayout (FrameLayout.java:270)
  at com.android.internal.policy.DecorView.onLayout (DecorView.java:789)
  at android.view.View.layout (View.java:24421)
  at android.view.ViewGroup.layout (ViewGroup.java:6440)
  at android.view.ViewRootImpl.performLayout (ViewRootImpl.java:4258)
  at android.view.ViewRootImpl.performTraversals (ViewRootImpl.java:3583)
  at android.view.ViewRootImpl.doTraversal (ViewRootImpl.java:2465)
  at android.view.ViewRootImpl$TraversalRunnable.run (ViewRootImpl.java:9305)
  at android.view.Choreographer$CallbackRecord.run (Choreographer.java:1339)
  at android.view.Choreographer$CallbackRecord.run (Choreographer.java:1348)
  at android.view.Choreographer.doCallbacks (Choreographer.java:952)
  at android.view.Choreographer.doFrame (Choreographer.java:882)
  at android.view.Choreographer$FrameDisplayEventReceiver.run (Choreographer.java:1322)
  at android.os.Handler.handleCallback (Handler.java:958)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loopOnce (Looper.java:205)
  at android.os.Looper.loop (Looper.java:294)
  at android.app.ActivityThread.main (ActivityThread.java:8177)
  at java.lang.reflect.Method.invoke
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:552)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:971)
Caused by java.lang.UnsupportedOperationException: Failed to resolve attribute at index 13: TypedValue{t=0x2/d=0x7f0405b6 a=-1}, theme={InheritanceMap=[id=0x7f15026dapp.pachli:style/TextSizeMedium], Themes=[app.pachli:style/TextSizeMedium, forced, app.pachli:style/DefaultTheme, forced, app.pachli:style/Theme.AppCompat.Empty, forced, android:style/Theme.DeviceDefault.Light.DarkActionBar, forced]}
  at android.content.res.TypedArray.getDrawableForDensity (TypedArray.java:1007)
  at android.content.res.TypedArray.getDrawable (TypedArray.java:991)
  at android.view.View.<init> (View.java:5706)
  at android.view.ViewGroup.<init> (ViewGroup.java:704)
  at android.widget.FrameLayout.<init> (FrameLayout.java:99)
  at android.widget.FrameLayout.<init> (FrameLayout.java:94)
  at android.widget.FrameLayout.<init> (FrameLayout.java:89)
```

**To Reproduce**

1. Launch Pachli in a Pixel C emulator
2. Login
3. Observe the crash

**Versions**

Pachli 1.0

**Comment by rmerren** [comment]
I am getting crash on login on a galaxy tab s4 after login. I am able to use the app on the same account from a phone.

**Comment by nikclayton** [comment]
@rmerren Thanks for the additional info. I can see this in the Play Console and it appears to be the same issue. I should have a fix for this very shortly, and 1.1 containing the fix before the end of the week.
